lib/OpenQA/WebAPI/Plugin/Gru.pm

```diff
@@ -19,25 +19,21 @@ package OpenQA::WebAPI::Plugin::Gru;
 use Mojo::Base 'Mojolicious::Plugin';
 
 use Minion;
-use Scalar::Util ();
 use DBIx::Class::Timestamps 'now';
 use Mojo::Pg;
 
-has [qw(app dsn)];
+has app => undef, weak => 1;
+has 'dsn';
```

Comment:
I know you didn't introduce this line, but I just want to note that I don't like cryptic abbreviations.

Reply:
The abbreviation `dsn` is actually a Perl DBI standard, and most Perl programmers know immediately what it means.

Reply:
This project is not targeting 'most Perl programmers' (whatever that means). It is targeting people who want to do QA work and likely even wish that project was written in a different language. Please keep that always in mind when doing something "Perlish". Likely it is not appropriate after all.

Reply:
This `dsn` abbreviation seems actually not Perl specific at all so I'm o.k. with it.